editor/src/uuiui/inputs/dropdown-number-input.tsx

```diff
@@ -0,0 +1,123 @@
+import type { CSSProperties } from 'react'
+import React from 'react'
+import { Icons } from '../icons'
+import { DropdownMenu, type DropdownMenuItem } from '../radix-components'
+import { NumberOrKeywordInput } from './new-number-or-keyword-input'
+import type {
+  CSSKeyword,
+  CSSNumber,
+  UnknownOrEmptyInput,
+} from '../../components/inspector/common/css-utils'
+import { isCSSNumber } from '../../components/inspector/common/css-utils'
+import { NO_OP } from '../../core/shared/utils'
+
+type DropdownNumberInputProps<T extends string> = {
+  testId: string
+  style?: CSSProperties
+  onSubmitValue: (value: UnknownOrEmptyInput<CSSNumber | CSSKeyword<T>>) => void
+  value: CSSNumber | CSSKeyword<T>
+  keywords: Array<{ label: string; value: CSSKeyword<T> }>
+  keywordTypeCheck: (keyword: unknown) => keyword is T
+}
+
+export function DropdownNumberInput<T extends string>(props: DropdownNumberInputProps<T>) {
```

Comment:
This could be called `NumberOrKeywordControl` (now that `NumberOrKeywordControl` has been renamed to `DEPRECATED_NumberOrKeywordControl`), because this is less of a number input and more of a CSS keyword input

Reply:
[`06df5ab` (#6161)](https://github.com/concrete-utopia/utopia/pull/6161/commits/06df5ab02168272c69a28100eefab4120515b836)